Mathlib/CategoryTheory/Sums/Basic.lean

```diff
@@ -203,15 +196,69 @@ theorem sum_obj_inr (F : A ⥤ B) (G : C ⥤ D) (c : C) : (F.sum G).obj (inr c)
   rfl
 
 @[simp]
-theorem sum_map_inl (F : A ⥤ B) (G : C ⥤ D) {a a' : A} (f : inl a ⟶ inl a') :
-    (F.sum G).map f = F.map f :=
+theorem sum_map_inl (F : A ⥤ B) (G : C ⥤ D) {a a' : A} (f : a ⟶ a') :
+    (F.sum G).map ((Sum.inl_ _ _).map f) = (Sum.inl_ _ _).map (F.map f) :=
   rfl
 
 @[simp]
-theorem sum_map_inr (F : A ⥤ B) (G : C ⥤ D) {c c' : C} (f : inr c ⟶ inr c') :
-    (F.sum G).map f = G.map f :=
+theorem sum_map_inr (F : A ⥤ B) (G : C ⥤ D) {c c' : C} (f : c ⟶ c') :
+    (F.sum G).map ((Sum.inr_ _ _).map f) = (Sum.inr_ _ _).map (G.map f) :=
   rfl
 
+section
+
+variable {F G: A ⊕ B ⥤ C}
+  (e₁ : Sum.inl_ _ _ ⋙ F ≅ Sum.inl_ _ _ ⋙ G)
+  (e₂ : Sum.inr_ _ _ ⋙ F ≅ Sum.inr_ _ _ ⋙ G)
```

Comment:
I think it's good style to include the explicit variables in each `def` (but not `theorem`), for readability. In this case one doesn't have to look far, but still!